pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdNamespaces.java

```diff
@@ -719,8 +719,14 @@ void run() throws PulsarAdminException {
         @Override
         void run() throws PulsarAdminException {
             String namespace = validateNamespace(params);
-            long sizeLimit = validateSizeString(limitStr);
+            long sizeLimit;
             long retentionTimeInSec;
+            try {
+                sizeLimit = validateSizeString(limitStr);
+            } catch (IllegalArgumentException e) {
+                throw new ParameterException(String.format("Invalid retention policy type '%s'. Valid formats are: %s",
```

Comment:
It would be better to specify the property with the invalid value, such as 'size limit'.

It's better to create a private method which build the error message, with property and valid values/examples as input